apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div
+    class="relative flex first:border-t border-b-[1px] border-neutral-200 hover:shadow-lg last:mb-0 p-4 w-full"
+    data-testid="cart-product-card"
+    v-if="orderItem.typeId !== 6"
+  >
+    <div class="relative overflow-hidden rounded-md w-[100px] sm:w-[176px] mr-4">
+      <SfLink :tag="NuxtLink" :to="localePath(orderGetters.getOrderVariationPath(order, orderItem) ?? '/#')">
+        <NuxtImg
+          class="h-auto border rounded-md border-neutral-200"
+          :src="addWebpExtension(orderGetters.getOrderVariationImage(order, orderItem)) || '/images/placeholder.png'"
+          :alt="'' || ''"
```

Comment:
does this have a purpose ?

Reply:
used same code component like in thank you page, should be removed or used properly i agree

Reply:
please fetch current version of branch